### DISCUSSION TITLE
Error shows up when I'm trying to renew my student benefits: [email] has already been used to obtain an academic discount on GitHub account(s) [username]

### DISCUSSION THREAD

**Discussion by nicholasxwang** [community/community#83792] [GitHub Education] [answered]
### Hi everyone,

I've been using GitHub's Student Developer Pack for a year and it expired June of 2023. I've not been able to renew it because of the error 
> [email] has already been used to obtain an academic discount on GitHub account(s) [username]

<img width="616" alt="Screenshot 2024-01-01 at 4 48 42 PM" src="https://github.com/community/community/assets/80929436/05ee726b-6e2e-4dee-bd68-dde94a7820de">

I used the same email the last time I registered. I can't use my school-provided email because our school disabled our inbox so I can't complete the email verification.

**Comment by nicholasxwang** [comment, marked as answer]
turns out i just had to switch browsers (from Arc to Edge) and ignore the error as it doesn't impact the application. I only thought it impacted my application because the green "Continue" button froze. 

If you encounter this error, just IGNORE it because it doesn't really mean anything.

**Reply by Andy-XO** [reply]
Will it be workong properly after changing the browser

**Reply by nicholasxwang** [reply]
yeah changing browsers fixes it.
